packages/react-console/package.json

```diff
@@ -33,7 +33,7 @@
   },
   "dependencies": {
     "@patternfly/patternfly": "4.59.1",
-    "@patternfly/react-core": "^4.75.9",
+    "@patternfly/react-core": "^4.75.10",
```

Comment:
Are you sure we should bump this? Typically, the @patternfly versions are kept in sync and updated with each milestone.

What's in @patternfly/react-core that is needed in the charts package?

Reply:
I am not sure. But if I don't bump this, it will fail the lint test on my computer. I will revert it.